charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml

```diff
@@ -161,10 +161,6 @@ data:
     {{- if .Values.global.gardenlet.config.featureGates }}
```

Comment:
There are some mentions left, that should be removed/adapted as well:
- https://github.com/gardener/gardener/blob/67582beb7138678b76f55c63339e67e76fc9b86c/docs/usage/managed_seed.md#L72
- https://github.com/gardener/gardener/blob/67582beb7138678b76f55c63339e67e76fc9b86c/docs/usage/shooted_seed.md#L59
- https://github.com/gardener/gardener/blob/5d4060aa8f5a315d49a931fe92f6246e142cbec3/example/55-managedseed-seedtemplate.yaml#L15
- https://github.com/gardener/gardener/blob/bd2763733c18fafee23959cb3fdc4dbf63f5853e/landscaper/pkg/gardenlet/apis/imports/validation/validation_test.go#L154